content/en/docs/languages/ruby/basics.md

```diff
@@ -334,6 +334,33 @@ resps.each do |r|
 end
 ```
 
+Non-blocking usage of the RPC stream can be achieved with multiple threads and 
+the `return_op: true` flag. When passing the `return_op: true` flag, the 
+execution of the RPC is deferred and an `Operation` object is returned. The RPC 
+can then be executed in another thread by calling the operation `execute` 
+function. The main thread can utilize contextual methods and getters such as 
+`status`, `cancelled?`, and `cancel` to manage the connection. This can be 
```

Comment:
s/the connection/the RPC

connections and RPCs aren't one to one, and these accessors actually don't give any direct control over the TCP connection